src/qibosoq/rfsoc_server.py

```diff
@@ -13,7 +8,7 @@
 from socketserver import BaseRequestHandler, TCPServer
 from typing import Dict, List
 
-from qick import QickSoc
+from qick import QickSoc  # type: ignore
```

Comment:
Qick again (maybe that's another thing we could do for Sho: typing his Python package)

Reply:
I think mypy complains because of the import, not really because of qick itself. I added the ignore for every qick import

Reply:
Yes, but the `import` statement has no problem on its own. Let me check the exact error...